src/code/z_lifemeter.c

```diff
@@ -508,7 +510,9 @@ void Health_UpdateBeatingHeart(PlayState* play) {
             interfaceCtx->beatingHeartOscillatorDirection = 0;
             if (!Player_InCsMode(play) && (play->pauseCtx.state == 0) && (play->pauseCtx.debugState == 0) &&
                 Health_IsCritical() && !Play_InCsMode(play)) {
```

Comment:
is this if really necessary at all? I looked at the functions quickly and it doesn't seem they're doing something else (at least I haven't seen anything obvious)

Reply:
Not sure what you’re saying here. Are you saying the original code should be changed? This looks to be making it so that the low health sound only plays during gameplay.

Reply:
I haven't tried to improve the way the original code was written yet (mainly because I would have no idea what I was doing and would probably break everything).
Maybe a big PR that does something like this in multiple places would be nice?